src/Nox.Reference.Data.World/Models/Countries/CountryInfo.cs

```diff
@@ -1,137 +1,155 @@
 ﻿using System.Text.Json.Serialization;
-using Nox.Reference.Abstractions;
-using Nox.Reference.Common;
 
-namespace Nox.Reference.Data.World;
+namespace Nox.Reference.Data.World.Models;
 
-internal class CountryInfo : ICountryInfo
+internal class CountryInfo
 {
-    [JsonPropertyName("id")]
-    public string Id { get; set; } = string.Empty;
+    [JsonIgnore]
+    public string Id => AlphaCode2;
 
-    [JsonPropertyName("name")]
-    public string Name { get; set; } = string.Empty;
-
-    [JsonPropertyName("code")]
-    public string Code { get; set; } = string.Empty;
-
-    [JsonPropertyName("languages")]
-    public IReadOnlyList<string> Languages { get; set; } = Array.Empty<string>();
+    [JsonIgnore]
+    public string Name => Names.CommonName;
 
-    [JsonPropertyName("names")]
-    [JsonConverter(typeof(NoxRefenceInfoJsonConverter<ICountryNames, CountryNamesInfo>))]
-    public ICountryNames Names { get; set; } = new CountryNamesInfo();
+    [JsonPropertyName("name")]
+    public CountryNamesInfo Names { get; set; } = new CountryNamesInfo();
 
-    [JsonPropertyName("topLevelDomains")]
-    public IReadOnlyList<string> TopLevelDomains { get; set; } = Array.Empty<string>();
+    [JsonPropertyName("tld")]
+    public IReadOnlyList<string> TopLevelDomains { get; set; } = new List<string>();
```

Comment:
hm, is this for de-serialization entity or for our normal one? I don't think that we want to rename fields for our entity this way

Reply:
{Entity}Info classes are only used for deserialization purpose